src/content/docs/infrastructure/host-integrations/host-integrations-list/sonarqube-monitoring-integration.mdx

```diff
@@ -21,314 +21,230 @@ After setting up our SonarQube integration, we give you a dashboard for your Son
 </figcaption>
 
 
-## Install the infrastructure agent [#infra-install]
+<Steps>
+    <Step>
+## Install the infrastructure agent [#infra-install]  
 
-You can install the infrastructure agent two different ways:  
+To use the SonarQube integration, you need to also [install the infrastructure agent](/docs/infrastructure/install-infrastructure-agent/get-started/install-infrastructure-agent-new-relic/) on the same host. The infrastructure agent monitors the host itself, while the integration you'll install in the next step extends your monitoring with SonarQube-specific data.
```

Comment:
I'm wondering if the boilerplate language for these should be something like "All our integrations first need the infrastructure agent to connect your data to NR." At least for this one, reframing it as something all OHIs use might make customers less 🙄 about all the different installs they need to do

Reply:
but written better obvs. not necessary but just something to think about